lib/run/options.js

```diff
@@ -110,6 +110,35 @@ var _ = require('lodash'),
         externalLoader(type, value, done);
     },
 
+    /**
+     * Custom method to auto parse CSV values
+     *
+     * @private
+     * @param {String} value - CSV field value
+     * @param {Object} context - Context of field value
+     * @param {Boolean} context.quoting - A boolean indicating if the field was surrounded by quotes.
+     * @returns {String|Number|Date}
+     */
+    csvAutoParse = function (value, context) {
+        if (context.quoting) {
```

Comment:
Make the if blocks consistent by returning the processed value directly, this will also let you remove `else` all over.